app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,9 @@
+class LikesController < ApplicationController
+  def create
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    @like = Like.new(user_id: @user.id, post_id: @post.id)
+
+    redirect_to user_post_path(@user, @post, @like)
+  end
```

Comment:
- [ ] Please check your like functionality it's not working kindly fix it check below :point_down:  my suggestion to fix this issue :slightly_smiling_face: 

```suggestion
  def create
    @user = current_user
    @post = Post.find(params[:post_id])
    @like = Like.new(user_id: @user.id, post_id: @post.id)

     if @like.save
      redirect_to user_post_path(@user, @post)
    end
  end
```

Reply:
- Well resolved